shopify-imagemagick@6.rb

```diff
@@ -5,8 +5,8 @@ class ShopifyImagemagickAT6 < Formula
   # ImageMagick site removes tarballs regularly which means we get issues
   # unnecessarily and older versions of the formula are broken.
   url "https://dl.bintray.com/homebrew/mirror/imagemagick%406-6.9.9-40.tar.xz"
-  mirror "https://www.imagemagick.org/download/ImageMagick-6.9.9-40.tar.xz"
-  sha256 "62f25d46bfbcffe00b84e167994d593868a9a47d4defc489a429cae1d6aab9f7"
+  mirror "https://download.imagemagick.org/ImageMagick/download/releases/ImageMagick-6.9.9-51.tar.xz"
+  sha256 "aa5f6b1e97bd98fbf642c47487531bea0faf675c728d01130b52d2b46849104a"
```

Comment:
Where did you get the SHA256 from?

Reply:
I ran `sha256sum ImageMagick-6.9.9-51.tar.xz` locally, but this should probably be obtained from an official source.

Reply:
Homebrew will give it to you, or `shasum -a256 ImageMagick-6.9.9-51.tar.xz`

Reply:
> but this should probably be obtained from an official source

Nah, what you did is perfect.